tests/Stecman/Component/Symfony/Console/BashCompletion/CompletionHandlerTest.php

```diff
@@ -148,6 +173,10 @@ protected function createHandler($commandLine, $cursorIndex = null)
      */
     protected function getTerms($handlerOutput)
     {
+        if (null === $handlerOutput) {
```

Comment:
`CompletionHandler::filterResults` and `CompletionHandler::runCompletion` currently return strings, but as the returned value [ends up being passed to `OutputInterface::write`](https://github.com/stecman/symfony-console-completion/blob/8ef3c83dac399929d68e5ba1cad3f12a6b4ed729/src/CompletionCommand.php#L73) which supports an array, this can be changed to return an array without any functional difference (the tests will just need to be updated).